### DISCUSSION TITLE
Blue Carbon Export

### DISCUSSION THREAD

**Discussion by tamaramegan** [Vizzuality/mangrove-atlas#295] [General]
Hello!
In the previous phase the blue carbon calculations were already made to show the values in the platform. As well, the files generated were saved as assets inside Google Earth Engine. There is one ImageCollection that has an image for each year, and each year has the different levels as separate bands (soil carbon, above ground carbon, below ground carbon, and total carbon all in CO2e units).   

To make the blue carbon available as a tiff we need to export these assets from GEE to the cloud bucket. 

The idea is to export only the Total Carbon band from each image (=year). These files are quite big and GEE splits them in several images. I made a test for 1996 and it exported 66 tiffs. The names generated automatically are very uninformative since they are generated automatically when GEE splits the images (we have no control on that). So, in order to give the user more information of the info inside each file, we were thinking of creating a grid and export the images in each grid cell adding the lat/long to the name of the file or something similar. So if a user wants data from the Caribbean, she/he knows which tiffs will have that info.
Something similar to what is done [here](https://data.globalforestwatch.org/documents/14228e6347c44f5691572169e9e107ad/explore) 

Let us know if that works for you. The link to download would take the users to the folder where all the tiffs fro one single year are stored.
Cheers!

**Comment by tamaramegan** [comment]
Following up with this comment, the export of files will need some more refinement, since GEE keeps splitting the files into smaller pieces after our approach. We will keep working on improving this workflow after the release.

As mentioned above, for now (release July 26th) the link will take the user to the bucket folder where all the files for one year are stored. If having one single file is the aim, we could explore zipping all the files together or merging into one single file, but that would not be available for Monday.

Would this work? 
Thanks!

**Reply by klongleywood** [reply]
Hi Tamara, I'll let @lhilarides weigh in too, but I think the tiled approach you describe should be fine.

**Comment by lyubovpencheva** [comment]
This is an old one.
